src/components/gataca-autoqr/gataca-autoqr.tsx

```diff
@@ -23,18 +57,20 @@ export class GatacaAutoQR {
   qr: GatacaQRWS | GatacaQR | GatacaSSIButton | GatacaSSIButtonWS;
 
   constructor() {
-    //TODO config ID is now an URL. Replace with an ID to ensure that all configs are stored in the place we desire (i.e: Studio)
     this.loading = true;
-    fetch(this.configId, {
+    fetch(this.configRepository + "/" + this.configId, {
       method: "GET",
       headers: {
         "Content-Type": "application/json",
       },
     })
       .then((response) => {
+        console.log("Got RESPONSE", response);
         response
           .json()
           .then((data: QRConfig) => {
+            console.log("Got DATA", data);
```

Comment:
Same here. If you want to keep into the component to use for development purposes comment it.